nucypher/config/node.py

```diff
@@ -61,6 +63,9 @@ class CharacterConfiguration(BaseConfiguration):
     DEFAULT_NETWORK_MIDDLEWARE = RestMiddleware
     TEMP_CONFIGURATION_DIR_PREFIX = 'tmp-nucypher'
 
+    # When we begin to support other threshold schemes, this will be one of the concepts that makes us want a factory.
```

Comment:
Seems related to #571? If so:
```suggestion
    # When we begin to support other threshold schemes, this will be one of the concepts that makes us want a factory - #571.
```